packages/yourloops/test/integration/mock/team.api.mock.ts

```diff
@@ -164,6 +165,46 @@ export const teamThree: Team = {
   }]
 }
 
+export const PatientNotification: ITeam = {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I don't understand this variable name... This is a team, named `PatientNotification`? I don't understand what it means :smile: